verilog/analysis/checkers/always_ff_non_blocking_rule.cc

```diff
@@ -48,50 +51,150 @@ absl::string_view AlwaysFFNonBlockingRule::Name() {
 }
 const char AlwaysFFNonBlockingRule::kTopic[] = "sequential-logic";
 const char AlwaysFFNonBlockingRule::kMessage[] =
-    "Use only non-blocking assignments inside \'always_ff\' sequential blocks.";
+    "Use blocking assignments, at most, for locals inside "
+    "'always_ff' sequential blocks.";
 
 std::string AlwaysFFNonBlockingRule::GetDescription(
     DescriptionType description_type) {
-  return absl::StrCat("Checks that there are no occurrences of ",
-                      "blocking assignment in sequential logic.");
+  return "Checks that blocking assignments are, at most, targeting "
```

Comment:
For rules with parameters, we add them in the description; please check out the other rules that provide configuration how that looks like.

This is then used to generate automatic documentation that then people can use to discover parameters: https://google.github.io/verible/lint.html